index.js

```diff
@@ -33,7 +33,7 @@ module.exports = class StylishReporter {
 
     function render(stats) {
       const opts = {
-        context: '/Users/powella/code/webpack-stylish/test/fixture',
+        context: process.cwd(),
```

Comment:
This should be `compiler.context`, as the actual context path can be (and frequently is) different than the cwd, and that's important.